### ISSUE TITLE
Lowering the minimum SDK requirement

### ISSUE THREAD

**Issue by traceur920** [zaneschepke/wgtunnel#25]
Hi, i've found this app really clean and ahead of the official Android Wireguard client, mainly as it allows to include and exclude apps at will without continuosly having to enable and disable the tunnel for that one problematic app.
Among my devices there are some who are a bit old but would benefit from this implementation a lot.
To be sure I've tested the build of the app down to API 21, allowing Android 5.0 - 8.0 devices to be included, and found no problems.
While the test was, for the 5.0 device, conducted on an emulator, the 8.0 was my daily driver and it seems to work fine.

So, can we lower the barrier a bit and set the minimum SDK requirement to API 26 or less?

**Comment by zaneschepke** [comment]
Hello and thank you for the valuable feedback! I will also do some testing. If all goes well, I will lower the minSdk requirements to at least 26.

**Comment by K4sum1** [comment]
I have an Android 5 phone I sometimes use, I could stick it on that for a while to test.

**Comment by zaneschepke** [comment]
> 

Do you need a build?

**Comment by K4sum1** [comment]
Well, I'm not in a rush, I don't use it that often and the current Wireguard app does what I need. However I'd be willing to test if a build gets released next time I use it.

**Comment by zaneschepke** [comment]
I did quite a bit of testing and was running into a lot of issues on the older devices (api 21-25). Eventually, I landed with getting everything working and comfortable on api 26. Seeing as that is the main request here, I'm going to publish a new build that supports api 26. If there is another request in the future to support even older devices, I will revisit the issues I was running into at that time.
